### DISCUSSION TITLE
Lighthouse station v2 vibrating frequently

### DISCUSSION THREAD

**Discussion by BEN-Frn** [bitcraze/discussions#1117] [Q&A]
hello all, My name is Ben and i am using crazyflie 2.1 and lighthouse positioning for my Masters project at University.

my query is to do with the lighthouse positioning systems. one of the base stations vibrates frequently whereas the other slightly 'hums'.
my question would be: should the base station vibrate frequently or does it require replacing?
i do believe this is affecting the stability of the crazyflie when observing it on the cfclient.

any advice on this query would be greatly appreciated 
Thanks,
Ben

**Comment by knmcguire** [comment]
Hi! That seems like a hardware fault to me. If you bought the lighthouse basestation from the Bitcraze store, send us an email at contact _at_ bitcraze.io with the order number and a description of the behavior of the basestation. We can arrange a replacement then.

**Reply by BEN-Frn** [reply]
Hello!
Thank you for the quick response

I have just sent the email over to bitcraze.io about the problem!
Hopefully hear from you guys soon!
